### DISCUSSION TITLE
Field names in dashboard (lists)

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#9045] [Feature Requests]
The dashboard lists could feature the fieldnames on top of the list.

The list could also feature a drag-able table, so that it do not look so messy if some field-contents are longer than others. This could also be good when displaying M2O in collections.

![image](https://user-images.githubusercontent.com/12813484/138415352-8c75ff0f-027b-4da1-9304-e0e12b87db68.png)

**Comment by benhaynes** [comment]
This has come up a lot. Th "repeater" style is very convenient as a default list in the App. That said, we will eventually add (back) support for a more traditional "table" style interface and panel. Not sure if this will be Core or Extension, but it's a good feature.

**Reply by rijkvanzanten** [reply]
@benhaynes The main difference is how you configure the two 🤔 For a table, you select the fields in order, for an inline list, you select the fields in order, but are allowed to add arbitrary characters in between. Theoretically speaking, we could simply extract the fields used in that template and render it as a table instead based on a boolean field 🤔

**Reply by benhaynes** [reply]
I think the bigger issue (IIRC) is how our `v-table` worked. It tried to be a little too smart (adjustable columns, etc), but the main issue was it didn't dynamically collapse down to the column's content width... which means you need all this other setup to make it work. Otherwise, we have a small status dot column takes up 200px, and a huge text limited to 200px.

I think we stick to the repeater style everywhere and add a "show as table" option... but that should use a new vue component that uses the actual `table` element. That will be simple enough to make happen.

Should be easy enough to get an external PR for.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
